nixos/modules/services/web-servers/bluemap.nix

```
@@ -267,6 +266,12 @@ in {
             You can achieve this through setting `services.bluemap.eula = true`
           '';
         }
+        { assertion = cfg.enableNginx -> cfg.host != null;
+          message = ''
+            You have enabled nginx to serve the bluemap webapp, but have not assigned a host
+            to bluemap through setting `services.bluemap.host = "hostname"`
```

Since `enableNginx` is `true` by default, maybe it'd better to say something like

```suggestion
            Nginx is enabled to serve the bluemap webapp, but you have not assigned a host
            to use. You can do this by setting `services.bluemap.host = "hostname"`.
```

Or something similar that doesn't imply the user did something

---

I liked this wording.

---

I ended up removing the assertion.